helper-cli/src/main/kotlin/commands/ListStoredScanResultsCommand.kt

```diff
@@ -70,7 +72,13 @@ internal class ListStoredScanResultsCommand : CommandWithHelp() {
 
         println("Searching for scan results of '${packageId.toCoordinates()}' in ${ScanResultsStorage.storage.name}.")
 
-        val scanResults = ScanResultsStorage.storage.read(packageId)
+        val scanResults = when (val readResult = ScanResultsStorage.storage.read(packageId)) {
+            is Success -> readResult.result
+            is Failure -> {
+                println("Could not read scan results: ${readResult.error}")
+                return 1
```

Comment:
The `ListLicensesCommand` returns  `2`in the error case.
I remember some discussion a while ago and I wrote parts of it into this commit message 8fd5a76.

Reply:
Done.